### ISSUE TITLE
Just a message to n0kk

### ISSUE THREAD

**Issue by jakadak** [n0kk/ahud#129]
Hi @n0kk,
I'm really sorry for contacting you this way.
I've got in touch with you a few weeks ago by one of your contributors about me accidentally attaching my personal information to one of the issues here, so I would like to thank you for deleting that issue back then. I really appreciate it!
Unfortunately, that issue is stored by Github's API in your public events feed, so it can still be easily googled. I talked with Github's support about that and they said they would need your permission to clear that public events feed.
Would that be okay for you? If yes, I think a simple reply to this issue would be just fine for Github's support.
Once again, I'm terribly sorry for taking your time and contacting you this untraditional way.
I look forward to hearing from you!
(PS: In case you'd like to see a screenshot of the conversation with the support, feel free to ask for it)

**Comment by n0kk** [comment]
Yeah, they can go ahead and delete the public events feed if it contains your personal information. If there was a way I can do it on my end, let me know—I don't really use GitHub a lot so I'm unfamiliar with the backend of the things on here.

Let me know when it's all done and I'll delete this issue when I'm available; I'm not sure I'm seeing another way for it to be deleted.

**Comment by jakadak** [comment]
Thanks,
You can delete this issue.
